RogueTech Core/GameTips/general.txt

```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
 The danger zones in a DropShip landing area are no joke, no matter what your fellow Mechwarriors say.
 Plan your lance according to the range & strategy you intend to use.
 A 'Mech needs an engine core to function. 
 An XL engine will save a lot of weight, but if you lose a Torso Side, you lose the 'Mech.
-Bring some streaks or pulse lasers to deal with those pesky Lights!
+Weapons with accuracy bonuses or evasion ignoring, like Pulse Lasers, are fantastic against fast-moving enemies.
 Watch your heat. Riding the redline will slow you down or blow you sky high.
-Riding the redline is dangerous... but fun.
+Riding the redline is dangerous...but fun.
 If you ever want to progress, you may need to punch above your lance's weight! 
 Choose your contracts carefully and you can get yourself some great salvage!
 Hitting an already overheated enemy with flamers or heat-generating weapons gives bonus damage!
-Buildings burn well. Buildings burn very well.
 It's harder for your pilots to see when it's dark or foggy.
 Trees are decent ablative armor, until they get set on fire.
-DHS Engine upgrades require a DHS Kit, which can be installed in the cooling slot.
+You can only install heatsinks that match the type of your heatsink kit.
 Repair costs of armor varies depending on type and Mechtech level. 
 Primitive Armour is cheap and easy to maintain. Hardened plates, not so much.
-ECM protects friendlies and degrades enemy sensors within its bubble.
+ECM Shield makes you harder to hit. ECM Jamming makes enemies more likely to miss.
 Active Probes boost your sensor check, improving the information you see about enemies at longer ranges.
 Stealth makes you harder to detect. It also provides a range based defense as well as extra defense the less you move!
 Narcs attach to a unit, providing visibility beyond sensor range and increased accuracy! This is opposed by enemy ECM, and lasts for a short duration.
 Tags increase your sensor check for tagged units, but this bonus degrades as they move from the position they were originally tagged.
 IFF jammers add a penalty to enemy sensor check's against a unit, making it harder to detect.
-UAV's allow you to do a quick scan of the Battlefield and Sensor Lock every Nearby Enemy.
-Melee combat in Roguetech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
+UAVs allow you to do a quick scan of the battlefield and sensor lock every nearby enemy.
+Melee combat in RogueTech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
 Railguns and Gauss rifles deal tremendous damage at equally tremendous ranges. Reach out and touch someone.
-Killing Rocket Launcher or RL units before they blow their load is tactically advisable.
+Use your sensors to check enemy loadouts! 'Mechs and vehicles with rocket launchers (RLs) can do a lot of damage.
```

Comment:
Feels like the old one is fine, and this should be a new one?
Maybe
`Better sensor rolls allow you to to check enemy loadouts!  You'll want to know if that truck is carrying a fluid gun, or something more sinister.`

Reply:
I felt that the innuendo implied was a little stale, personally, but obviously it's not my work so if people want it kept then that's fine.

Reply:
Reverted in commit https://github.com/BattletechModders/RogueTech/pull/4298/commits/3ccb47eec110b546d94250908361e9a526950a4e